doc/developer/testdrive.md

```diff
@@ -787,6 +787,13 @@ once one record matches, the following must all match.  There are permitted to b
 topic after the matching is complete.  Note that if the topic is not required to have `partial-search`
 elements in it but there will be an attempt to read up to this number with a blocking read.
 
+#### `kafka-verify-topic [sink=... | topic=...] [await-value-schema=false] [await-key-schema=false]`
```

Comment:
OOC can this be used for the progress topic as well?

Reply:
If you specify the topic name rather than specifying the sink, yes